lib/pbench/agent/tool_meister.py

```diff
@@ -552,6 +671,8 @@ def start_tools(self, data):
         failures = 0
         tool_cnt = 0
         for name, tool_opts in sorted(self._tools.items()):
+            if name in self.persist_tools:
+                continue
```

Comment:
This is done so often, I almost wonder if we should just track persistent/transient tools separately and join the two sets when we really need them all... but I guess it's not worth worrying about right now.